### DISCUSSION TITLE
Order by computed value

### DISCUSSION THREAD

**Discussion by testerez** [prisma/prisma#15724] [Q&A] [answered]
Hello,
I'm using Prisma with a postgres DB.
My use case: I want to order invoices by subtotal. An invoice has `invoiceItems` and the subtotal is calculated as `sum(invoiceItem.unitPrice * invoiceItem.quantity)`.
I also want to be able to apply various filters on this query and paginate with `take`/`skip`.

So far, I've found that I could use raw queries but then I kind of loose most benefits from Prisma.

Or I'd be happy to use a postgres view to pre-compute the `subtotal`. But it looks like I'd then wouldn't be able to use Prisma migrations anymore, which I see as a deal breaker.

Any idea on the best approach to implement this?

**Comment by pantharshit00** [comment, marked as answer]
Hello,

Currently this is not possible as you have mentioned. We have an outstanding feature request related to this here: https://github.com/prisma/prisma/issues/5848

I would suggest you to leave your feedback there and add +1 to the top comment to help prioritise this.

**Reply by testerez** [reply]
Thanks @pantharshit00 , I wish I could just flag a model so that it's ignored by the migration. That way, I could easily create views for computed values.

**Reply by pantharshit00** [reply]
Maybe this upcoming proposal is also helpful here: https://github.com/prisma/prisma/issues/15074#issuecomment-1282419200. Otherwise unfortunately you will need to wait here.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
